src/rust/ide/src/test.rs

```diff
@@ -38,6 +49,229 @@ pub mod mock {
+        pub fn foo_method_parameter2() -> suggestion_database::Argument {
+            suggestion_database::Argument {
+                name          : "param1".to_owned(),
+                repr_type     : "Number".to_owned(),
+                is_suspended  : false,
+                default_value : None,
+            }
+        }
+
+        pub fn bar_method_parameter() -> suggestion_database::Argument {
+            suggestion_database::Argument {
+                name          : "this".to_owned(),
+                repr_type     : "Other".to_owned(),
+                is_suspended  : false,
+                default_value : None,
+            }
+        }
+
+        pub fn suggestion_entry_foo() -> suggestion_database::Entry {
+            suggestion_database::Entry {
+                name      : "foo".to_owned(),
+                module    : module::QualifiedName::from_segments("Std",&["Base"]).unwrap(),
+                self_type : Some("Base".to_owned()),
+                arguments : vec![foo_method_parameter(),foo_method_parameter2()],
+                return_type   : "Any".to_owned(),
+                kind          : suggestion_database::EntryKind::Method,
+                scope         : suggestion_database::Scope::Everywhere,
+                documentation : None
+            }
+        }
+
+        pub fn suggestion_entry_bar() -> suggestion_database::Entry {
+            suggestion_database::Entry {
+                name      : "bar".to_owned(),
+                module    : module::QualifiedName::from_segments("Std",&["Other"]).unwrap(),
+                self_type : Some("Other".to_owned()),
+                arguments : vec![bar_method_parameter()],
+                return_type   : "Any".to_owned(),
+                kind          : suggestion_database::EntryKind::Method,
+                scope         : suggestion_database::Scope::Everywhere,
+                documentation : None
+            }
+        }
+    }
+
+    #[derive(Clone,Debug)]
+    pub struct Unified {
+        pub logger        : Logger,
+        pub project_name  : String,
+        pub module_path   : model::module::Path,
+        pub graph_id      : double_representation::graph::Id,
+        pub suggestions   : HashMap<suggestion_database::EntryId,suggestion_database::Entry>,
+        pub context_id    : model::execution_context::Id,
+        pub parser        : parser::Parser,
+        code              : String,
+        id_map            : ast::IdMap,
+        metadata          : crate::model::module::Metadata,
+        root_definition   : double_representation::definition::DefinitionName,
+    }
+
+    impl Unified {
+        pub fn set_inline_code(&mut self, code:impl AsRef<str>) {
+            let method = self.method_pointer();
+            //self.code = format!("{}.{} = {}",method.defined_on_type,method.name,code.as_ref())
+            // FIXME [mwu] should be as above but definition lookup doesn't handle properly implicit
+            //             module name
```

Comment:
Do we plan to fix that? Do we have an issue for it?

Reply:
Actually the problem was in the test code, will be resolved.